samples/listctrl/listtest.cpp

```diff
@@ -1204,13 +1205,21 @@ void MyListCtrl::OnChecked(wxListEvent& event)
 {
     LogEvent(event, "OnChecked");
 
+    if ( IsVirtual() ) {
```

Comment:
Usual formatting nitpick (here and below):
```suggestion
    if ( IsVirtual() )
    {
```